Stepts taken to reproduce MGXlab/pvogs_function repository

1. we run into the following error:
(pvogs) seb@Ras:~/Fonda/notebook_reproduction/pvogs_function/results/.logs/pre_process$ cat process_annotations.log
/home/seb/Fonda/notebook_reproduction/pvogs_function/.snakemake/conda/5e32c691/lib/python3.8/site-packages/pandas/_testing.py:24: FutureWarning: In the future `np.bool` will be defined as the corresponding NumPy scalar.
  import pandas._libs.testing as _testing
Traceback (most recent call last):
  File "workflow/scripts/process_annotations.py", line 4, in <module>
    import pandas as pd
  File "/home/seb/Fonda/notebook_reproduction/pvogs_function/.snakemake/conda/5e32c691/lib/python3.8/site-packages/pandas/__init__.py", line 180, in <module>
    import pandas.testing
  File "/home/seb/Fonda/notebook_reproduction/pvogs_function/.snakemake/conda/5e32c691/lib/python3.8/site-packages/pandas/testing.py", line 5, in <module>
    from pandas._testing import (
  File "/home/seb/Fonda/notebook_reproduction/pvogs_function/.snakemake/conda/5e32c691/lib/python3.8/site-packages/pandas/_testing.py", line 24, in <module>
    import pandas._libs.testing as _testing
  File "pandas/_libs/testing.pyx", line 10, in init pandas._libs.testing
  File "/home/seb/Fonda/notebook_reproduction/pvogs_function/.snakemake/conda/5e32c691/lib/python3.8/site-packages/numpy/__init__.py", line 305, in __getattr__
    raise AttributeError(__former_attrs__[attr])
AttributeError: module 'numpy' has no attribute 'bool'.
`np.bool` was a deprecated alias for the builtin `bool`. To avoid this error in existing code, use `bool` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.bool_` here.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations

2. The error of using the deprecated np.bool is not easy to fix.
    Searching the repository I find no relevant mentions of it.
    Might be a problem with one of the included packages.
    from the stacktrace I can't even discern where the deprecated np.bool is used
    the rule ("parse_annotations") that causes the error tries to execute "python workflow/scripts/process_annotations.py" in the shell.
    the file mentioned ("workflow/scripts/process_annotations.py") does not contain the pattern "bool"

3. I could try to rerun this with an updated version of pandas. That seems to have worked (at least solved the np.bool error). Executing the workflow locally at the moment:
    Given the description in github, the workflow should download its own data
    The rule "hmmsearch_transeq" is taking a lot of time; running for 1 hour now; I killed the workflow after 2 hours of no progress on the rule


